rules/cc_side_outputs.bzl

```diff
@@ -84,15 +87,14 @@ def _cc_compile_different_output(target, ctx, extension, flags):
             "{}.{}.{}".format(target.label.name, source_file.basename, extension),
         )
         outputs.append(output_file)
-
-        #output_file_tmp = ctx.actions.declare_file(output_file.basename + ".tmp")
+        is_c_or_cc = _is_c_or_cc(source_file)
```

Comment:
Why is this separated from the one location it's being used 7 lines from here?

Reply:
No reason. I think it was used multiple places and then I deleted them. Inlined it!